java/acorn-config/src/main/java/com/datasqrl/ai/config/AcornAgentConfiguration.java

```diff
@@ -33,19 +35,29 @@ public class AcornAgentConfiguration {
   public static final String API_PREFIX = "apis";
   public static final String CONVERTER_PREFIX = "converter";
 
-  public static final String CLIENT_FUNCTIONS_KEY = "client_functions";
   public static final String PROMPT_KEY = "prompt";
-  public static final String LOCAL_FUNCTIONS_KEY = "local_functions";
+  public static final String FUNCTIONS_KEY = "functions";
   public static final String CONTEXT_KEY = "context";
 
   Configuration baseConfiguration;
   Configuration modelConfiguration;
   List<RuntimeFunctionDefinition> toolFunctions;
+  ModelObservability observability;
+
 
-  private RuntimeFunctionDefinition loadLocalFunction(String functionClassName) {
-    if (!functionClassName.contains(".")) {
```

Comment:
@mbroecheler with the removal of local functions, I can't add the datacharts functions anymore.
`  "functions": "datacharts",`
causes
```Caused by: java.lang.IllegalArgumentException: Could not locate user defined function: com.datasqrl.ai.function.builtin.datacharts```
I don't understand how you intended to load the UI functions